ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/usertags/cucushift-installer-rehearse-azure-ipi-usertags-workflow.yaml

```diff
@@ -1,9 +0,0 @@
-workflow:
-  as: cucushift-installer-rehearse-azure-ipi-usertags
```

Comment:
I think we can keep the workflow, so that in future, once usertag on azure get supported, we can use it directly.